src/hooks/useInterval.ts

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export function useInterval(callback: Function, interval = 0) {
+    const savedCallback = React.useRef<Function>();
+
+    React.useEffect(() => {
+        savedCallback.current = callback;
+    }, [callback]);
+
+    React.useEffect(() => {
+        const id = setInterval(() => {
+            savedCallback.current?.();
+        }, interval * 1000);
```

Comment:
Why do you translate from milliseconds to seconds?

Reply:
I guess it is simpler to enter a value, safer to not have the opportunity to call function 1000 time in a second and I can't imagine the situation where you need to send a message more often than one tome in a second